scipy/stats/_multivariate.py

```diff
@@ -5579,10 +5578,18 @@ class uniform_direction_gen(multi_rv_generic):
     :math:`S^2`.
 
     Alternatively, the object may be called (as a function) to fix the `dim`
-    parameter, return a "frozen" `random_direction` random variable:
-
-    >>> rv = uniform_direction(5)
-
+    parameter, return a "frozen" `uniform_direction` distribution. Here
+    we create a ``uniform_direction`` with ``dim=3`` and draw 5 samples.
+    The samples are then arranged in an array of shape 5x3.
+
+    >>> uniform_sphere_dist = uniform_direction(3)
```

Comment:
See gh-13778, gh-13863.

```suggestion
    >>> rng = np.random.default_rng()
    >>> uniform_sphere_dist = uniform_direction(3)
```

PR to add document this in https://docs.scipy.org/doc/scipy/dev/missing-bits.html#missing-bits would be welcome.